Dockerfile-base

```diff
@@ -0,0 +1,19 @@
+FROM node:8
+
+ENV HOME "/home/xpub"
+
+RUN mkdir -p ${HOME}
+
+WORKDIR ${HOME}
+
+# Build yarn offline cache
+
+RUN yarn config set yarn-offline-mirror /npm-packages-offline-cache && \
+  yarn config set prefer-offline true && \
+  yarn config set yarn-offline-mirror-pruning true
+
+RUN git clone https://github.com/elifesciences/elife-xpub.git && \
```

Comment:
right, this is not using `xpub-collabra`

Reply:
Yes, it used to, but it didn't make sense since the specific tag we used was being built by us and being used by us.

Given the location of it, should we change it to clone `xpub-collabra`?